### DISCUSSION TITLE
OnlyM Not Opening - Windows 11 Pro

### DISCUSSION THREAD

**Discussion by joshncampbell** [AntonyCorbett/OnlyM#451] [Q&A]
We've been having an issue that started about a month ago in which opening the executable "visually" does nothing. I've tried running as an Admin and removed all Anti-virus in case it was causing an issue. So far I still can't get it to open. Any suggestions are very much appreciated.

**Comment by AntonyCorbett** [comment]
@joshncampbell Thanks for your report. Some questions/things to try:

1. When you launch OnlyM, does the process load (i.e. is there an "OnlyM" entry in Task Manager)?
2. Does the log file reveal any issues? (See the wiki for the location of logs.)
3. Perhaps you can try deleting the settings folder (%AppData%\OnlyM) and relaunching

**Reply by joshncampbell** [reply]
Attached is the log file from today when trying to run it. This might be a key to the issue:
System.Windows.Markup.XamlParseException: The invocation of the constructor on type 'OnlyM.Windows.MainWindow' that matches the specified binding constraints threw an exception.
---> System.DllNotFoundException: Unable to load DLL 'SQLite.Interop.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)

[log-20220601.txt](https://github.com/AntonyCorbett/OnlyM/files/8816017/log-20220601.txt)

**Reply by joshncampbell** [reply]
I forgot to answer your first question: No, it does not show in the Task Manager. 

I did delete the settings folder and tried relaunching but that didn't get it working. The log I previously sent was created after I deleted the settings folder.

**Reply by AntonyCorbett** [reply]
Do the `SQLite.Interop.dll` and `System.Data.SQLite.dll` files exist in the OnlyM installation folder, please?

**Reply by joshncampbell** [reply]
Yes, they are both present in "C:\Program Files (x86)\OnlyM". See attached. 
![Installation Directory](https://user-images.githubusercontent.com/61886081/171479738-6c1dec5b-d067-4c45-8dd8-d2f50853f887.png)

**Reply by AntonyCorbett** [reply]
Thanks. Have you reinstalled OnlyM?

**Reply by joshncampbell** [reply]
Yes. I did a complete uninstall, removed the directory for it in %AppData%, and then installed it again. Same behavior after the fresh install.

**Reply by joshncampbell** [reply]
BTW about 2 weeks ago it started working. We're not sure why or what may have changed...but right now the program is working again. If it happens again I'll report it in case we can find the "why". Thanks!

**Comment by khvideo** [comment]
We are having a problem when we try to display a video or picture.  The image does not display on any of our monitors.  We have uninstalled and reinstalled the program.  We have tried a different version of OnlyM.  We have also removed a Windows update in an effort to get the video to display.  Do you have any suggestions?

**Reply by AntonyCorbett** [reply]
@khvideo Sorry for the late response. Have you specified a media monitor or a "windowed" output option in the OnlyM settings?
